src/fu-engine.c

```diff
@@ -2073,8 +2073,8 @@ fu_engine_publish_release(FuEngine *self, FuRelease *release, GError **error)
 	GInputStream *stream = fu_release_get_stream(release);
 
 	/* send to passimd, if enabled and running */
-	if (passim_client_get_version(self->passim_client) != NULL &&
-	    fu_engine_config_get_p2p_policy(self->config) & FU_P2P_POLICY_FIRMWARE) {
+	if (fu_engine_config_get_p2p_policy(self->config) & FU_P2P_POLICY_FIRMWARE &&
+	    passim_client_get_version(self->passim_client) != NULL) {
```

Comment:
I'm surprised this works; passim_client_get_version() is just a getter -- passim_client_load() is the thing that's going to start the daemon I think.

Reply:
Oh, I didn't test that part of it, I was hoping the reporter could!